test/spec/token.js

```diff
@@ -62,6 +62,154 @@ define(function(require) {
       });
     });
 
+    it('returns id_token using sessionToken with issuer', function (done) {
+      return oauthUtil.setupFrame({
+        oktaAuthArgs: {
+          url: 'https://auth-js-test.okta.com',
+          clientId: 'NPSfOkH5eZrTy8PMDlvx',
+          redirectUri: 'https://auth-js-test.okta.com/redirect',
```

Comment:
it won't. I'll change it to be a different domain.

edit: sorry @rchild-okta, accidentally deleted your comment - you asked if the redirectUri will usually be on the okta domain.